catroidTest/src/org/catrobat/catroid/test/utils/Reflection.java

```diff
@@ -127,8 +127,6 @@ public ParameterList(Object... parameters) {
 				objectClass = int.class;
 			} else if (objectClass == Long.class) {
 				objectClass = long.class;
-			} else if (objectClass == Short.class) {
-				objectClass = short.class;
 			}
```

Comment:
http://pmd.sourceforge.net/pmd-4.3.0/rules/controversial.html#AvoidUsingShortType

@cheidenreich
Please undo these changes and exclude the rule. The physics engine (and box2d) is using shorts for collision filtering.